### PR TITLE
US001 SQL migration scripts

### PR DESCRIPTION
**Pull request by ghost**
Screenshot 2024-08-26 at 16 12 22
Screenshot 2024-08-26 at 16 13 16

### CODE REVIEW

File: src/main/resources/db/migration/V1__Create_capability_table.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE capability (
+    capabilityId INT PRIMARY KEY,
+    capabilityName VARCHAR(255) NOT NULL
```
Comment (Alicja-Kainos) [inline src/main/resources/db/migration/V1__Create_capability_table.sql]: 255 seems to long for capability name, I would limit it to 25 chars

File: src/main/resources/db/migration/V2__Create_band_table.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE band (
+    bandId INT PRIMARY KEY,
+    bandName VARCHAR(255) NOT NULL
```
Comment (Alicja-Kainos) [inline src/main/resources/db/migration/V2__Create_band_table.sql]: also too long varchar, I would limit it to 25 chars

File: src/main/resources/db/migration/V3__Create_job_roles_table.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE job_roles (
+    jobRoleId INT PRIMARY KEY,
+    roleName VARCHAR(255) NOT NULL,
```
Comment (Alicja-Kainos) [inline src/main/resources/db/migration/V3__Create_job_roles_table.sql]: Please count chars in "Principal workday integration solution architect for smart product" -> I don't know longer role :D And limit to sth around the count

File: src/main/resources/db/migration/V3__Create_job_roles_table.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    location VARCHAR(255),
```
Comment (Alicja-Kainos) [inline src/main/resources/db/migration/V3__Create_job_roles_table.sql]: max 25? check the longest town/city name in the word ;)

File: src/main/resources/db/migration/V2__Create_band_table.sql (hunk 2 above)
Comment (ghost) [reply src/main/resources/db/migration/V2__Create_band_table.sql]: Fixed

File: src/main/resources/db/migration/V3__Create_job_roles_table.sql (hunk 3 above)
Comment (ghost) [reply src/main/resources/db/migration/V3__Create_job_roles_table.sql]: Fixed

File: src/main/resources/db/migration/V3__Create_job_roles_table.sql (hunk 4 above)
Comment (ghost) [reply src/main/resources/db/migration/V3__Create_job_roles_table.sql]: Fixed - changed to enum

**Review by Alicja-Kainos — APPROVED** [review]

**Review by patrykpkainos — APPROVED** [review]

**Pull request merged**